apps/www/_alternatives/supabase-vs-firebase.mdx

```diff
@@ -0,0 +1,71 @@
+- Realtime - [changes in your data will be streamed directly to your application](https://supabase.com/docs/reference/dart/subscribe).
+- Auth - [a simple to integrate auth system and SQL based rules engine](https://supabase.com/auth).
+- Functions - [javascript and typescript functions that deploy out globally](https://supabase.com/edge-functions).
+- Storage - [hosting images, videos, and pdfs easily](https://supabase.com/storage).
+
+## How are they similar?
+
+Both Firebase and Supabase are based on the idea of bringing a superior developer experience to databases. With both platforms you can spin up a new project from directly inside the browser without the need to download any extra tools or software to your machine. Both platforms come with a useful dashboard UI for debugging your data in realtime, which is especially useful for fast iterations when in development.
+
+Both Firebase and Supabase have invested heavily in client side libraries so you can communicate with your database directly from the client. Firebase has their [Firebase Javascript SDK](https://github.com/firebase/firebase-js-sdk) and Supabase has [supabase-js an isomorphic client](https://github.com/supabase/supabase-js/) that can be used both on the client also on the server in a node-js environment.
+
+## How are they different?
+
+Firebase and Supabase differ in several ways. The main one being that Firebase is a document store, whereas Supabase is based on PostgreSQL - a relational, SQL-based database management system. But there are also some other important differences.
+
+### Open Source
+
+Supabase is fully open source. This means that along with the hosted cloud platform, you can also take the Supabase stack and host it inside your own cloud or run it locally on your machine. The primary benefit here is no vendor lock in, which can be a frustrating problem for some Firebase users.
+
+### Pricing
+
+[Firebase charges for reads, writes and deletes](https://firebase.google.com/pricing), which can lead to some unpredictability, especially in the early stages of a project when your application is in heavy development. Supabase on the other hand [charges based on the amount of data stored](https://supabase.com/pricing), with breathing room for unlimited API requests and an unlimited number of Auth users.
+
+### Scaling
+
+One scaling limitation you might face when using Firestore - is the 1 document per second limitation on writes. It can be hard to architect around this issue for users who want to be able to sustain high write loads from a single client. Supabase is built on Postgres which is a well battle tested at scale, to help scale the number of connections. Every deployment comes with an instance with PgBouncer, a [Postgres connection pooler perfect for use in Serverless computing](https://supabase.com/docs/guides/database/connecting-to-postgres#connection-pool) environments.
```

Comment:
"is the 1 document per second limitation on writes"

If we call this out, we should link to the section in their docs.

"It can be hard to architect around this issue for users who want to be able to sustain high write loads from a single client."


Remove this - just let the limitation stand and don't tell people what they will find hard